CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 **New Features**
 
 - Hot-swap themes: No need to restart the Medusa after changing the theme ([#4271](https://github.com/pymedusa/Medusa/pull/4271))
+- Added ability to manually change episode quality ([#4658](https://github.com/pymedusa/Medusa/pull/4658))
```

Comment:
I wouldn't call it a new feature...

Reply:
It was never possible to only change the quality of an episode... so it definitely is IMO.

Reply:
But we can also move it to improvements, it's the same for me.

Reply:
I feel like improvements would be a better description.